vcr/root/root.go

```diff
@@ -116,6 +117,7 @@ func NewCmdRoot(f cmdutil.Factory, version, buildDate, commit string, updateStre
 	cmd.AddCommand(debugCmd.NewCmdDebug(f))
 	cmd.AddCommand(deployCmd.NewCmdDeploy(f))
 	cmd.AddCommand(instanceCmd.NewCmdInstance(f))
+	cmd.AddCommand(logCmd.NewCmdLog(f))
```

Comment:
please add log command as a sub command to instance, as logging is for instances